src/mparticle-instance-manager.ts

```diff
@@ -480,11 +489,14 @@ function mParticle() {
     };
 }
 
-var mparticleInstance = new mParticle();
+// Can we get rid of this and just rename the class to mparticleInstanceManager?
+const mparticleInstance = new mParticle();
```

Comment:
i don't know about getting rid of it, but renaming it per my agreement above would be fine.